spec/lib/rdstation/account_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+RSpec.describe RDStation::Account do
+  let(:client) do
+    described_class.new(authorization: RDStation::Authorization.new(access_token: 'access_token'))
+  end
+
+  let(:endpoint) { 'https://api.rd.services/marketing/account_info' }
+
+  let(:headers) do
+    {
+      'Authorization' => 'Bearer access_token',
+      'Content-Type' => 'application/json'
+    }
+  end
+
+  let(:error_handler) do
+    instance_double(RDStation::ErrorHandler, raise_error: 'mock raised errors')
+  end
+
+  before do
+    allow(RDStation::ErrorHandler).to receive(:new).and_return(error_handler)
+  end
+
+  describe '#info' do
+    it 'calls retryable_request' do
+      expect(client).to receive(:retryable_request)
```

Comment:
What about to use "have received" as in [spec/lib/rdstation/account_spec.rb:53](https://github.com/ResultadosDigitais/rdstation-ruby-client/pull/86/files#diff-4871f2225a1eb771b8e55c348bda2ee9b88aad916fd5cc510f2aa2a24f118f16R53) to standardize the spec, or use "receive" and change the other spec though.

Reply:
I agree with you @gil27 , but at this moment to apply these changes it'll be necessary some refactor to avoid other bad practices.

To use `have_received` it'll be necessary to make a stub for the subject, which is not a good idea.

Maybe we can do this in another PR. 

What do you think?